index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Spazeir Mart</title>
+    <link href="style.css" rel="stylesheet" type="text/css" />
+    <link href="https://spazeir-ui.netlify.app/CSS/components.css" rel="stylesheet" type="text/css" />
+</head>
+
+<body>
+    <div class="hero-header">
+        <p class="brand-name">Spazeir Mart</p>
+        <p class="hero-category-name">Grocery</p>
+        <input class="input-textbox" type="text" placeholder="Search here..." name="text" />
+        <button class="button primary-yellow">Login</button>
+        <button class="button primary-yellow">Cart</button>
+        <button class="button primary-yellow">Wishlist</button>
+    </div>
+    <div class="categories">
+        <div class="child">
```

Comment:
child is very generic, it can mean different things in different context.. it should have a classname like category-card or something